### ISSUE TITLE
Why the giou_loss initial value is so low

### ISSUE THREAD

**Issue by jty928284127** [ultralytics/yolov5#431]
Why the giou_loss initial value is so low

When I use other frameworks for training, the initial loss value is several thousand,；
But in yolov5  the initial loss value is only 0.1

thanks

**Comment by abhiagwl4262** [comment]
Hey @glenn-jocher. I am little surprised to see some results :- 

I am adding a log and not able to figure out the reason why val loss has become "NAN" for few epoches and then recovered even with larger learning rates.

![Screenshot from 2021-02-23 23-46-11](https://user-images.githubusercontent.com/22645080/108888396-6e3e8c00-7631-11eb-8715-988fa538e020.png)

Can this be possible ? And is there something in the repo that is making this happen ?

**Comment by glenn-jocher** [comment]
@abhiagwl4262 anything's possible, especially with poor hyperparameters.

I would use default training settings, and if your losses diverge then increase warmup epochs and reduce LR.
